### DISCUSSION TITLE
Custom Domain SSL Issue

### DISCUSSION THREAD

**Discussion by x3221** [deta/cloud-docs#665] [Help] [answered]
I added a custom domain to my micro yesterday. When I try to access the website it shows that the certificate expired in 2021 and returns `{"message":"Internal Server Error"}`. 

Not sure what went wrong, do I just have to wait for deta to issue a valid certificate?

**Comment by arguskao** [comment]
I have the same problem.

**Reply by x3221** [reply]
It's probably an issue with your CAA configuration.

**Comment by x3221** [comment, marked as answer]
Was a CAA configuration issue.
